gobblin-api/src/main/java/gobblin/configuration/ConfigurationKeys.java

```diff
@@ -475,6 +475,15 @@
   public static final String COMPACTION_OVERWRITE_OUTPUT_DIR = COMPACTION_PREFIX + "overwrite.output.dir";
   public static final boolean DEFAULT_COMPACTION_OVERWRITE_OUTPUT_DIR = false;
   public static final String COMPACTION_JARS = COMPACTION_PREFIX + "jars";
+  public static final String COMPACTION_DATA_COMPLETENESS_VERIFICATION_THREAD_POOL_SIZE =
```

Comment:
These configs are specific to the MRCompactor. Please declare them in the class that uses them.